src/ruleset.hpp

```diff
@@ -73,8 +79,10 @@ struct ruleset {
     std::unordered_map<std::string_view, exclusion::input_filter::ptr> input_filters;
     // Rules are ordered by ID
     std::unordered_map<std::string, rule::ptr> rules;
-    // Collections are ordered by rule.type
-    std::unordered_map<std::string, std::vector<rule::ptr>> collections;
+
+    // Collections
```

Comment:
It would be useful to continue documenting the meaning of the key of the map.

Reply:
It seems I had good intentions, but was betrayed by my memory.